measurements.py

```diff
@@ -0,0 +1,34 @@
+''' Definition of the measurement class for the CU Hackathon Project: WheresMySat
+
+Parent class meas records a measurement to be fed into the Kalman Filter
+
+All children classes of meas contain any extraneous data and the system dynamics
+
+Author:Connor Johnstone
+'''
+
+#Import Libraries
+import numpy as np
+from numpy import linalg as LA
+
+class Meas():
```

Comment:
ABC and @abstract method for the following: 
`residual` 
`calculate` (or dynamics or something)